packages/firestore/src/platform_rn/rn_platform.ts

```diff
@@ -15,22 +15,36 @@
  * limitations under the License.
  */
 
-import { BrowserPlatform } from '../platform_browser/browser_platform';
 import { base64 } from '@firebase/util';
 
-// Implements the Platform API for ReactNative.
-export class ReactNativePlatform extends BrowserPlatform {
-  readonly base64Available = true;
+// ReactNative-specific platform implementation that provides its own Base64
+// encoding.
+// The exports in this class must match the exports in '../platform/platform' as
+// they are bundled with the browser build during the Rollup build.
```

Comment:
These aren't bundled with the browser build--it's the react native, build, right?

Reply:
That's correct. The comment has been removed.